core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -69,10 +69,10 @@
     <div ng-cloak class="navbar-header float-right">
       <ul class="nav oppia-navbar-nav oppia-navbar-profile-admin">
         <li>
-          <ul class="nav oppia-navbar-tabs-admin oppia-admin-page-dropdown" uib-dropdown>
+          <ul class="nav oppia-navbar-tabs-admin oppia-admin-page-dropdown" uib-dropdown ng-class="{'open' : ctrl.dropdownMenuisActive}">
```

Comment:
Are you sure you have tested these changes? I was expecting `$ctrl` here

Reply:
My bad, changed
surprisingly it worked before :|